src/loaders/css-module-decorator-loader/loader.ts

```diff
@@ -1,6 +1,8 @@
+import webpack = require('webpack');
```

Comment:
I believe we can use the import/from syntax, but I've run into issues where importing from a module typed with `export =` triggers a compile error (and, of course, I can't find any examples right now). Because of that, I tried to standardize on using import/require for modules that export a function and import/from for ES6-like modules.